CMD_HELP.md

```diff
@@ -18,7 +18,7 @@ annotate subcommand options:
  -e,--error-report-location <arg>   Error report filename (including path)
  -f,--filename <arg>                Mutation filename
  -h,--help                          shows this help document and quits.
- -i,--isoform-override <arg>        Isoform Overrides (mskcc or uniprot)
+ -i,--isoform-override <arg>        Isoform Overrides. Options: `mskcc` (preferred) or uniprot (legacy)
```

Comment:
@inodb Not sure if we should call uniprot "legacy"?

Reply:
I like that idea yea 👍